### ISSUE TITLE
Calendar Matrix issue

### ISSUE THREAD

**Issue by angelnikolov** [kurkle/chartjs-chart-matrix#16]
Hello, thanks for the nice plugin!
### Background
I am trying to create a calendar matrix, similar to one of the samples in this Repository.
This is the example I've created.
https://codepen.io/darkysharky/pen/qBqvGEy?editors=0010

### Issue
What I need is day names on the y axis, and weeks on the X axis.
Everything seems to work properly, but the only issue I stumble upon is the y axis names. As you can see, some are duplicated, some are missing and the order is not kept at all.
Basically, I want to have the days from Monday to Sunday on the left and the weeks on the bottom, so I can quickly see at what day and what week something has occurred.

**Comment by kurkle** [comment]
You changed to moment adapter, and removed the `parser` option.
For moment, I think the correct option might be `parser: 'd'` (for the y-scale).

https://codepen.io/kurkle/pen/PobgPqp

**Comment by kurkle** [comment]
Another note, it might be easier to use `category` scale for the day, but I don't remember why I did not do that originally.

**Comment by angelnikolov** [comment]
@kurkle Yeah tried that, but then there were other issues, like for example, data for `Sunday` from week to going into week 1, because there, we don't have data for `Sunday`. I will try your suggestion with the parser. Thank you!

**Comment by angelnikolov** [comment]
@kurkle I used your idea with the parser and it works nicely, however I still get the same issue I keep getting with the previous implementation.
https://codepen.io/darkysharky/pen/eYBoeOL?editors=0010
![image](https://user-images.githubusercontent.com/2675259/111292048-7433f580-8650-11eb-8f10-246525fd26ba.png)
![image](https://user-images.githubusercontent.com/2675259/111292109-84e46b80-8650-11eb-9b16-6152fa8c9b7b.png)

On the screenshot you see how you can hover on the boxes next to each other which should show 20 Feb and 21 Feb, but they somehow show 20 Feb and 14 Feb. It seems like the whole bottom row is moved with one box to the right. I thought the reason for that was because I used 2.9.4, but I tried it with 3 now and the same issue occurs.
I will appreciate any ideas, I am empty of them :(
